src/app/public/modules/vertical-tabset/vertical-tab.component.ts

```diff
@@ -98,19 +107,20 @@ export class SkyVerticalTabComponent implements OnInit, OnDestroy {
     this.tabsetService.addTab(this);
   }
 
-  public ngOnDestroy() {
+  public ngOnDestroy(): void {
     this._mobileSubscription.unsubscribe();
+    this.tabsetService.destroyTab(this);
```

Comment:
This is the one addition to this file. The rest should be just style changes 👆 .